ui/store/actions.test.js

```diff
@@ -1199,30 +1199,36 @@ describe('Actions', () => {
     });
   });
 
-  describe('#setRpcTarget', () => {
+  describe('#setActiveNetwork', () => {
     afterEach(() => {
       sinon.restore();
     });
 
-    it('calls setRpcTarget', async () => {
+    it('calls setActiveNetwork', async () => {
       const store = mockStore();
 
-      background.setCustomRpc.callsFake((_, __, ___, ____, cb) => cb());
+      const setCurrentNetworkStub = sinon.stub().callsFake((_, cb) => cb());
 
-      _setBackgroundConnection(background);
+      background.getApi.returns({
+        setActiveNetwork: setCurrentNetworkStub,
+      });
+      _setBackgroundConnection(background.getApi());
 
-      await store.dispatch(actions.setRpcTarget('http://localhost:8545'));
-      expect(background.setCustomRpc.callCount).toStrictEqual(1);
+      await store.dispatch(actions.setActiveNetwork('http://localhost:8545'));
+      expect(setCurrentNetworkStub.callCount).toStrictEqual(1);
```

Comment:
Nit: It might be worthwhile confirming that this is called with the correct value

Reply:
done here: https://github.com/MetaMask/metamask-extension/pull/17421/commits/ccfc91787ab01012269c2d2bd97fa9b3c0de5ace